recipes/cad1/README.md

```diff
@@ -0,0 +1,91 @@
+# The 1st Cadenza Challenge
+Cadenza challenge code for the 1st challenge.
+
+For more information about the Cadenza Challenge please visit https://cadenzachallenge.com
+
+Cadenza tutorials are now available in https://cadenzachallenge.com.
+The tutorials introduce the Cadenza installation, how to interact with Cadenza metadata,
+and also provide examples of baseline systems and evaluation tools.
+
+## Data structure
+
+To download data, please visit [here](https://mab.to/zU7TS8jJelkoD).
+The data is split into three packages:
+`clarity_CEC2_core.v1_0.tgz` [28 GB],
```

Comment:
These could be a bullet list, and could even link to the files directly...

```suggestion
* [`clarity_CEC2_core.v1_0.tgz`](<path_to_file>) [28 GB],
```

Reply:
Hosting service doesn't provide direct links hence whey these can't be included.